checklists/apim_checklist.en.json

```diff
@@ -139,6 +139,7 @@
             "waf": "Reliability",
             "guid": "beae759e-4ddb-4326-bf26-47f87d3454b6",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = ( sku.name == "Premium" and  isnotnull(properties.additionalLocations)) | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes